ngraph/src/ngraph/op/softplus.hpp

```diff
@@ -0,0 +1,52 @@
+//*****************************************************************************
+// Copyright 2017-2020 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+
+#pragma once
+
+#include "ngraph/node.hpp"
+#include "ngraph/op/op.hpp"
+
+namespace ngraph
+{
+    namespace op
+    {
+        namespace v4
+        {
+            /// \brief A Self Regularized Non-Monotonic Neural Activation Function
+            /// f(x) =  x * tanh(log(exp(x) + 1.))
```

Comment:
`log` -> `ln`

Reply:
And may be `f(x) =  tanh(ln(exp(x) + 1.))` instead of `f(x) =  x * tanh(ln(exp(x) + 1.))`. Where did you get that the value must be multiplied with `x`?

Reply:
Comments fixed.